other-packages/babel-plugin-export-metadata/src/index.js

```diff
@@ -22,7 +22,11 @@ const replaceExportDefault = template(`
 
 const getFilename = state => {
   const filename = get(state, 'file.opts.filename')
-  return filename && path.relative(process.cwd(), filename)
+  let filePath = filename && path.relative(process.cwd(), filename)
```

Comment:
Probably `return filename && path.posix.relative(process.cwd(), filename)` would be enough. Unsure what this does exactly, but in previous versions of Docz often changing to path.posix was enough.

Reply:
hi @patricksevat 
I'm not so familiar with `path` module. I searched what can `path.posix.relative` do and seems that it could not tranform '\\` to '/` in win32 platform (https://github.com/nodejs/node/issues/13887). And currently there is no `path.posix` code in docz. This pr's main target is tranform '..\\a\\b\\c' to '../a/b/c' in win32.